### ISSUE TITLE
Noto Sans Thaana font no longer following Faruma style (?)

### ISSUE THREAD

**Issue by davelab6** [notofonts/thaana#4]
A user wrote in:

> I am writing to address a significant issue regarding the recent update to the Dhivehi Noto Sans Thaana font. Unfortunately, the current version of the font does not meet the established standards for professional documents and deviates from the proper representation of the Dhivehi language.
> 
> The updated font styles are highly unconventional and do not align with the traditional and official use of Dhivehi script. This is a departure from previous versions of Dhivehi Noto Thaana Sans, which were more aligned with the official standards and the government’s preferred font for formal communication.
> 
> In particular, the official font used in government documents and standard letters is Faruma. The current iteration of Dhivehi Noto Sans does not replicate this standard, which is critical for maintaining consistency and professionalism in our documents.
> 
> Given that we rely on Google Docs for writing in Thaana and this is the only available font for us, the recent changes render the font practically unusable for our needs. Therefore, I kindly urge you to revert to the previous version of the Dhivehi Noto Sans Thaana font or to make the necessary adjustments to better align it with the Faruma font and official standards.

**Comment by davelab6** [comment]
Sadly this contradicts https://github.com/notofonts/thaana/issues/1 and https://medium.com/kodefly/is-noto-sans-thaana-relevant-29a8307fb6cb

We have no way to contact the user, so hoping they find this issue. If we have no more info, I suggest closing as "WAI" in 6 months.

**Comment by davelab6** [comment]
I wonder if @pathumego @inerds @aliharis have any insights :)

**Comment by inerds** [comment]
Hey @davelab6,

I haven’t been following the updates recently, but I was surprised to find that the font style has completely changed to have very sharp edges. I believe this is what the user is referring to. Almost all Thaana fonts traditionally have very rounded corners, so sharp edges like those in the current Noto Sans Thaana are quite rare.

<img width="1059" alt="Screenshot 2024-07-31 at 11 26 15" src="https://github.com/user-attachments/assets/005af172-6cfc-4700-8558-955d36e6e8fb">

**Comment by davelab6** [comment]
Okay :) To recap, here's v2:

<img width="1299" alt="Noto Sans Thaana v2 thin weight" src="https://github.com/user-attachments/assets/aae655da-ca5d-4196-8982-94094d9c9a9e">
<img width="1299" alt="Noto Sans Thaana v2 regular weight" src="https://github.com/user-attachments/assets/be08b8f3-4589-4408-a52d-d5d3c6363de5">
<img width="1299" alt="Noto Sans Thaana v2 black weight" src="https://github.com/user-attachments/assets/68a006e7-acb2-46d0-83ac-154f41c7446d">

In https://github.com/notofonts/thaana/issues/1#issuecomment-1641793380 you said 

> i wouldn't consider current style [v2] as acceptable. Styling is definitely not as par the Noto standards.

Personally I am an advocate for post-modern global culture, and I don't see any issue with making 'sharp edge' designs that are not traditional. The 'loopless' style of Thai might be similar, in a way. I'm happy for the world to have both. 

So, perhaps v3 should be remain released as Noto Sans Thaana, and v2 should be re-released as Noto **Serif** Thaana, rather than abandoned completely. 

What about v2's styling "is definitely not as par the Noto standards"?

**Comment by inerds** [comment]
In V2, the styling wasn’t quite up to the standard when compared to other Noto fonts or even other Thaana fonts that are available. Unfortunately, the newly introduced styling in V3 feels more like a rough, handwritten font, which might not align with the expectations for a polished typeface.

That said, I’m not opposed to the idea of having both sans and serif versions of Noto Thaana. These could certainly add value, provided that the styling is refined to meet the quality of other Noto scripts.

To help with this, I’d like to share some resources that might inspire further development:
- [thaana.com](https://thaana.com/) – A great source of modern Thaana typography, featuring primarily display fonts.
- [hassanhameed.com/thaana-fonts](https://hassanhameed.com/thaana-fonts) – author of several popular Thaana fonts, most of which are in a formal style and available for free.
- [akurutype.com](https://akurutype.com/) – A marketplace for various typefaces.
- [theloopcraft.com/raajjetype](https://theloopcraft.com/raajjetype) – A commercial type foundry.

I’m also here to clarify any questions type designers might have.

**Comment by davelab6** [comment]
Thank you so much @inerds - I really value your expertise here. I'm still not sure what wasn't slightly below expectations, could you give just 1 visual comparison example and explanation? I'll try to secure 2025 budget to work on this fully.

**Comment by inerds** [comment]
@davelab6  To clarify things better and ensure we’re on the same page, would it be possible to schedule a quick call? I think it would help me explain what I’m seeing and provide more context

**Comment by marcoT0425** [comment]
I think it's OK. Because it is a sans font. Comparing with the latin letters

**Comment by K1ngfish3r** [comment]
The font in its current state is quite unusable to send to my lecturers. It is hard to read as part of a paragraph, and requires a large font size (around 14-15pts) to be slightly legible since this font tends to be on the smaller size.

Please note that the examples below are with font-size 14 for noto sans thaana, and font-size 12 for faruma
The comma needs to facing up rather than down,

![Image](https://github.com/user-attachments/assets/ef45058d-0656-41e0-a230-0136254fd8ab)

![Image](https://github.com/user-attachments/assets/ae728dcc-5574-478c-9d5e-cac13339829c)

The full stop is lower than it needs to be, it seems to be taking the tail end of the letters to place the full stop rather than the middle.

![Image](https://github.com/user-attachments/assets/f6fdf5f9-c99a-4339-b976-0b5bd31c916f)

![Image](https://github.com/user-attachments/assets/34fd3276-bc9f-4a5d-a127-3bcbbdef66ce)

The lack of a professional font for dhivehi typewriting in the google suite is a loss and I don't wish to move to Microsoft Word If I can help it.

**Comment by marcoT0425** [comment]
I have a new idea. I can split it to two fonts. One is Noto Sans Thaana and the another one is Noto Sans Thaana Traditional (v2.000)

**Comment by K1ngfish3r** [comment]
> I have a new idea. I can split it to two fonts. One is Noto Sans Thaana and the another one is Noto Sans Thaana Traditional (v2.000)

That is perfectly fine as the current font option is highly illegible/not suitable for an educational setting
Would the latest version here be the default across all platforms? It would honestly be best if Noto Sans Thaana Traditional is the default whenever Dhivehi language is in use. Or Noto Serif Thaana as @davelab6 called it.

**Comment by marcoT0425** [comment]
Yes, they are two different families. Like Noto Sans Thai (Modern) and Noto Sans Thai Looped (Looped). Noto Sans Thaana can be used in a more relaxed manner, but Noto Sans Thaana Traditional is based on MV Boli, with more rounded corners.

**Comment by K1ngfish3r** [comment]
>Noto Sans Thaana can be used in a more relaxed manner

tbh, can we call `Noto Sans Thaana` as `Noto Sans Thaana Modern`? and leave v2.000 as the normal version? I tried finding a similar font as to how the latest version feels and this is one (close one atleast in feeling, not the best example) https://fonts.google.com/specimen/Snowburst+One
Imagine trying to read a thesis using the previous font. Some letters are too long whilst others are too short, it does not follow formal writing standards used in normal dhivehi typesetting and I doubt people can write like this normally unless they are trying really hard to do so.
If MV Boli is the Comic Sans of our language (which is quite disingenuous to Comic Sans, since Comic Sans actually has it's uses), I'm not sure how to feel about the current version of Noto Sans Thaana.

**Comment by marcoT0425** [comment]
Sure

**Comment by MohEze** [comment]
> Sure

Hello? any progress on this?
